core/che-core-api-core/src/test/java/org/eclipse/che/api/core/util/CompositeLineConsumerTest.java

```diff
@@ -0,0 +1,134 @@
+package org.eclipse.che.api.core.util;
+
+import org.eclipse.che.api.core.util.lineconsumer.ConsumerAlreadyClosedException;
+import org.mockito.Mock;
+import org.mockito.testng.MockitoTestNGListener;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+import java.nio.channels.ClosedByInterruptException;
+
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+/**
+ * @author Mykola Morhun
+ */
+@Listeners(value = {MockitoTestNGListener.class})
+public class CompositeLineConsumerTest {
+
+    @Mock
+    private LineConsumer lineConsumer1;
+    @Mock
+    private LineConsumer lineConsumer2;
+    @Mock
+    private LineConsumer lineConsumer3;
+
+    private CompositeLineConsumer compositeLineConsumer;
+
+    private LineConsumer subConsumers[];
+
+    @BeforeMethod
+    public void beforeMethod() throws Exception {
+        subConsumers = new LineConsumer[] { lineConsumer1, lineConsumer2 };
+
+        for (LineConsumer lineConsumer : subConsumers) {
+            doNothing().when(lineConsumer).writeLine(anyString());
```

Comment:
I think that it is default behavior. Do we need to declare that in additionally?